### PR TITLE
Interactable Card V2

### PR DESCRIPTION
**Pull request by lilyvc**
This PR creates an Interactable Card component in lab.

`InteractableCardGroup` is currently wip. Feedback is welcome.
Currently it only handles single select and it is using a button element.

### CODE REVIEW

File: packages/lab/src/interactable-card/InteractableCard.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   */
+  onChange?: (event: MouseEvent<HTMLButtonElement>) => void;
+  /**
+   * Styling variant; defaults to "primary".
+   */
+  variant?: "primary" | "secondary";
+  /**
+   * Value of card (for selectable use case).
+   */
+  value?: string | number;
```
Comment (joshwooding) [inline packages/lab/src/interactable-card/InteractableCard.tsx:45]: This should match the value prop of something like Checkbox or ToggleButton

File: packages/lab/src/interactable-card/InteractableCardGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      select,
+      isSelected,
+      disabled,
+    }),
+    [select, isSelected, disabled]
+  );
+
+  return (
+    <InteractableCardGroupContext.Provider value={contextValue}>
+      <FlexLayout
```
Comment (joshwooding) [inline packages/lab/src/interactable-card/InteractableCardGroup.tsx:110]: Typically we avoid using our layout components in our components

File: packages/lab/src/interactable-card/InteractableCardGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  } = props;
+
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-toggle-button-group",
+    css: interactableCardGroupCss,
+    window: targetWindow,
+  });
+
+  const groupRef = useRef<HTMLDivElement>(null);
```
Comment (joshwooding) [inline packages/lab/src/interactable-card/InteractableCardGroup.tsx:71]: Unused?

File: site/docs/components/card/examples.mdx [hunk 4]
```diff
@@ -10,46 +10,30 @@ data:
 
 <LivePreviewControls>
 
-    <LivePreview componentName="card" exampleName="Default" >
-      ### Default
-    </LivePreview>
+    <LivePreview componentName="card" exampleName="InteractableCardExample" displayName="Interactable Card">
+      ### Interactable card
```
Comment (joshwooding) [inline site/docs/components/card/examples.mdx:14]: All of these headers should be H2s now

File: packages/lab/stories/interactable-card/interactable-card.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+          <H3>Cryptocurrency</H3>
+        </StackLayout>
+        <Text>Link your crypto to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+  </InteractableCardGroup>
+);
+
+export const InteractableCardGroupRadio: StoryFn<typeof InteractableCard> = (
+  args
```
Comment (lilyvc) [inline packages/lab/stories/interactable-card/interactable-card.stories.tsx:225]: This would basically be how you implement selectable card pattern

**Comment by lilyvc** [comment]
I think the card has to be a div not a button because you can't put a radio button inside a button (accessibility test)

**Pull request closed without merging**